requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 coverage
 flake8
+ipdb
```

Comment:
OK I'll remove it. 
This sounds like one of those etiquette things... is adding a new dev dependency here a general no-no? Or is there another reason for it?